agagd/agagd/settings/prod.py

```diff
@@ -2,14 +2,20 @@
 
 from base import *
 
-DEBUG = False
+if os.getenv('DEBUG') == 'true':
+  DEBUG = True
+else:
+  DEBUG = False
+
 TEMPLATE_DEBUG = DEBUG
 ADMIN_ENABLED = False
 
-_user = os.environ['AGAGD_USER']
-_password = os.environ['MYSQL_PASS']
-_key = os.environ['SECRET_KEY']
-_dbname = os.environ['APP_DB_NAME']
+_user = os.getenv('AGAGD_USER', '')
```

Comment:
This is probably the most controversial change here, as it touches existing code. Elaboration on my reasoning, by providing defaults and behavior switches based on env variables, we can eventually merge the 'prod' and 'dev' settings modules into one settings file that is configured/controlled more fully from environment. This obviously lends itself naturally to containerized deployment, but even without that it creates a clearer configuration/code boundary. (see the 12-factor app or other modern productionization methodology of your choice, I guess)

The one possible concern is the default setting of `SECRET_KEY`, since you could theoretically/feasibly deploy the checked-in key if you're not careful. If that's a major concern given your deployment strategies, I can refactor that part to be more careful. (it causes some problems, e.g. in the dockerfile where we run `manage.py collectstatic`)

Reply:
I think we should use caution here, and we should avoid the issues of having a default value for `SECRET_KEY`.

Reply:
Noted and fixed.